03_launch_mgmt_cluster.sh

```diff
@@ -161,6 +161,19 @@ function launch_baremetal_operator() {
     popd
 }
 
+function launch_kind() {
+  reg_ip="$(docker inspect -f '{{.NetworkSettings.IPAddress}}' registry)"
+  cat <<EOF | kind create cluster --name kind --image=kindest/node:"${KUBERNETES_VERSION}" --config=-
+  kind: Cluster
+  apiVersion: kind.x-k8s.io/v1alpha4
+  containerdConfigPatches:
+  - |-
+    [plugins."io.containerd.grpc.v1.cri".registry.mirrors."192.168.111.1:5000"]
+      endpoint = ["http://${reg_ip}:5000"]
```

Comment:
Is there really a need to go with the Docker IP instead of the 192.168.111.1 here ?